src/helpers/accessControl/rules/structureRules.ts

```diff
@@ -51,4 +51,9 @@ export default async function structureRules(
   can([action.read], ressource.conseillersRuptures, {
     structureId: user?.entity.oid,
   });
+  can([action.read, action.update], ressource.users, {
+    'entity.$id': {
+      $in: conseillersIds,
```

Comment:
préciser rôles $in conseiller - oid unique par collection même si ca devrait jamais arriver